index.js

```diff
@@ -36,21 +64,21 @@ const defaultTarget = '_blank'
 const defaultRel = ['nofollow', 'noopener', 'noreferrer']
 const defaultProtocols = ['http', 'https']
 
+/**
+ *
+ * @param {Options[keyof Options]} value
+ * @param {Element} node
```

Comment:
We might be able to type this. Perhaps you need to explicitly pass a `key` as a parameter (and then also maybe pass the whole options?).
Something along these lines?:

```suggestion
 * If this is a value, return that.
 * If this is a function instead, call it to get the result.
 *
 * @template {keyof Options} Key
 * @param {Options[Key]} value
 * @param {Element} node
 * @returns {Exclude<Options[Key], Function>}
```